cmake/opentelemetry-proto.cmake

```diff
@@ -229,31 +229,41 @@ endif()
 
 include_directories("${GENERATED_PROTOBUF_PATH}")
 
+unset(OTELCPP_PROTO_TARGET_OPTIONS)
+if(CMAKE_SYSTEM_NAME MATCHES "Windows|MinGW|WindowsStore")
+  list(APPEND OTELCPP_PROTO_TARGET_OPTIONS STATIC)
+elseif(NOT DEFINED BUILD_SHARED_LIBS OR BUILD_SHARED_LIBS)
+  list(APPEND OTELCPP_PROTO_TARGET_OPTIONS SHARED)
+else()
+  list(APPEND OTELCPP_PROTO_TARGET_OPTIONS STATIC)
+endif()
+
+list(APPEND OTELCPP_PROTO_TARGET_OPTIONS)
```

Comment:
Not sure if I am missing something, but how are we using `OTELCPP_PROTO_TARGET_OPTIONS`

Reply:
Sorry, it's a mistake after merging.